app/grandchallenge/datatables/static/js/datatables/list.mjs

```diff
@@ -53,12 +40,3 @@ $(document).ready(() => {
         },
     });
 });
-
-$("#ajaxDataTable").on("init.dt", () => {
-    // This is a work-around to get the table to resize properly on extra-large Bootstrap viewport
-    setTimeout($("#ajaxDataTable").DataTable().columns.adjust, 1000);
-});
-
-$(window).resize(() => {
```

Comment:
No need to account for resizing anymore: this was to help the datatable responsive components pack in more of the columns in the dropdown.